src/Cookie.php

```diff
@@ -73,11 +81,16 @@ public function __construct(
         $path = null,
         $secure = false,
         $httpOnly = false,
-        \DateTime $expires = null
+        \DateTime $expires = null,
+        $requireValidation = true
```

Comment:
I don't really like the idea of having optional validation. Have it or not IMO. So I would just deprecate the exception and bypass the validation manually.